lib/component-repository/package.json

```diff
@@ -28,11 +28,12 @@
     "eslint": "7.32.0",
     "eslint-plugin-mocha": "9.0.0",
     "eslint-plugin-node": "11.1.0",
+    "faker": "^5.5.3",
```

Comment:
Please use this lib instead:
https://www.npmjs.com/package/@faker-js/faker

Why?
https://fossa.com/blog/npm-packages-colors-faker-corrupted/

Reply:
Replaced!